.travis/test-coverage.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
this is actually something that could be rewritten in Go and put as a standalone command in `gonum/tools` :)
(this script is using many UNIX commands...)

Reply:
File an issue for that? It can live here and be used Gonum-wide.

Reply:
done: https://github.com/gonum/tools/issues/5